app/components/Chat.vue.ts

```diff
@@ -7,7 +7,7 @@ import { getPlatformService } from '../services/platforms';
 import { CustomizationService } from '../services/customization';
 import url from 'url';
 import electron from 'electron';
-import { ICustomizationSettings } from "../services/customization/customization-api";
+import { ICustomizationSettings } from '../services/customization/customization-api';
```

Comment:
We have webpack configured so you can reference things relative to the `app` directory.  It makes things easier when moving files around.  So this can just be: `import { ICustomizationSettings } from 'services/customization/customization-api';`

There are lots of places we don't do this, but we are trying to opportunistically change them as we touch files.